tests/settings/e2e/steps/currency-display-settings.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import { When, Then} from 'cucumber';
+
+const CONVERTED_CURRENCY_WALLET_DISPLAY_GENERAL = '//*[@class="WalletSummaryCurrency_currencyRate"]';
+const DISPLAY_BALANCE_OTHER_CURRENCYS_TOGGLE_STATUS_OFF = '//*[@class="SimpleSwitch_switch SwitchOverrides_switch"]';
+const DISPLAY_BALANCE_OTHER_CURRENCYS_TOGGLE_STATUS_ON = '//*[@class="NormalSwitch_component SimpleSwitch_root SwitchOverrides_root SimpleSwitch_checked SwitchOverrides_checked"]';
+const SELECT_CURRENCY_DROPDOWN = '//*[@label="Select currency"]';
+
+When(/^I open currency selection dropdown$/, function() {
+  return this.waitAndClick(SELECT_CURRENCY_DROPDOWN);
+});
+
+When(/^I select "([^"]*)" as the selected displayed currency$/, function(currency) {
+  return this.waitAndClick(`//*[contains(text(), "${currency}")]`);
+});
+
+Then(/^I should see "([^"]*)" displayed beside wallet balance$/, function(ticker) {
+  return this.client.waitForVisible(`//*[@class="WalletSummaryCurrency_currencyRate" and text()="${ticker}"]`);
+});
+
+When(/^I toggle the button (on|off) to change if I want to see my ada balance in other currency's$/, function(switchStatus) {
+  let selector = DISPLAY_BALANCE_OTHER_CURRENCYS_TOGGLE_STATUS_ON;
+  if(switchStatus === 'on'){
+    selector = DISPLAY_BALANCE_OTHER_CURRENCYS_TOGGLE_STATUS_OFF;
+  }
+  return this.waitAndClick(selector);
+});
+
+Then(/^the currency selection box is (hidden|visible)$/, function(state) {
+  let invisibility = false;
+  if (state === 'hidden') {
+    invisibility = true;
+  }
+  return this.client.waitForVisible(SELECT_CURRENCY_DROPDOWN, null, invisibility);
+});
+
+Then(/^The wallet summary screen (does|does not) show ada balance in other currency's placeholder$/, function(switchstatus) {
+  let invisible = true;
```

Comment:
You could write this in one line `const invisible:boolean = switchstatus !== 'does'`

The return of a comparison is always a boolean.

Reply:
I like this solution very much. It condenses 4 lines of code in to 1 awesome 👍